manual/src/tutorials/polyvariant.etex

```diff
@@ -29,9 +29,9 @@ let f = function `On -> 1 | `Off -> 0 | `Number n -> n;;
 List.map f [`On; `Off];;
 \end{caml_example}
 "[>`Off|`On] list" means that to match this list, you should at
-least be able to match "`Off" and "`On", without argument.
+least be able to match "`Off" and "`On", without arguments.
```

Comment:
I'm tempted to say this was correct as-is - polymorphic variant constructors can only accept a single argument (same with the change on L34). @Octachron?

Reply:
Yes, polymorphic variant constructors takes at most one argument (which might be a tuple).

Reply:
In that case, should it be "without **an** argument"?

Reply:
It's technically grammatical with or without the "an", but "without an argument`" idiomatically reads more strongly as an instruction not to shout at the compiler 🙂 Although, to be honest, now that I've read it that way, I can't "un-read" that interpretation of _argument_ 😂

Alternatively, ```least be able to match "`Off" and "`On" as constant constructors.``` or some such?

Reply:
Ah, I do see it could be read that way. I'll just leave it how it was originally, then.